update minimum Kubernetes version to 1.19

zr-msft opened the pull request:
relates to #108

bridgetkromhout commented:
Related: https://github.com/openservicemesh/osm-docs/issues/135

shashankram reviewed (commented):
Looks good, just need to resolve conflicts.

zr-msft commented:
thank you @shashankram. i rebased and fixed the conflicts.

@bridgetkromhout per @shashankram this should be good to merge. Please take a look and merge if you're ok with ti
